src/remediations/remediations.queries.js

```diff
@@ -517,6 +518,97 @@ exports.getIssueSystems = function (id, tenant_org_id, created_by, issueId) {
     });
 };
 
+// Return a paginated, sorted list of distinct systems for a remediation plan
+exports.getPlanSystems = async function (
+    remediation_plan_id,
+    tenant_org_id,
+    created_by,
+    column = 'display_name',
+    asc = true,
+    filter = null,
+    limit = 50,
+    offset = 0
+) {
+    const { Op, s: { literal, col, cast }, issue, issue_system, remediation } = db;
+
+    // Fetch distinct system ids for the plan scoped by tenant/user
+    const distinctSystemIds = (await issue_system.findAll({
+        attributes: [[db.s.fn('DISTINCT', db.s.col('issue_system.system_id')), 'system_id']],
```

Comment:
This should be expressed the way it is on line 674, with the import for DISTINCT moved to the top of the file.